sns/marshaler.go

```diff
@@ -8,18 +8,20 @@ import (
 	"github.com/ThreeDotsLabs/watermill/message"
 )
 
+// todo: check if it can be renamed
```

Comment:
@Mariscal6 is `UUID` special in any way? or we could rename it to `_watermill_uuid` to avoid conflicts?

Reply:
No it is not special, we can modify it